provision/acc_provision/templates/provision-config.yaml

```diff
@@ -67,7 +67,8 @@ registry:
 #kube_config:
   # ovs_memory_limit: "20Gi"            # override if needed, default is "1Gi"
   # reboot_opflex_with_ovs: "false"     # override if needed, default is "true"
-
+  # snat_operator:
+        # disable_periodic_snat_global_info_sync: True
```

Comment:
This example does not seem to be consistent with the overrides configuration that you have here:
https://github.com/noironetworks/acc-provision/pull/505/files#diff-a4df73be61e65d3f6ba506e9f158411a0e411551e7a3baca74b597a0b63bad94R49

which does not put it in a snat_operator section. Which one is correct, this or that?